mobile/examples/super_resolution/app/src/main/java/ai/onnxruntime/example/superresolution/SuperResPerformer.kt

```diff
@@ -0,0 +1,61 @@
+package ai.onnxruntime.example.superresolution
+
+import ai.onnxruntime.OnnxJavaType
+import ai.onnxruntime.OrtSession
+import ai.onnxruntime.OnnxTensor
+import ai.onnxruntime.OrtEnvironment
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import java.io.InputStream
+import java.nio.ByteBuffer
+import java.util.*
+
+internal data class Result(
+    var outputBitmap: Bitmap? = null
+) {}
+
+internal class SuperResPerformer(
+    private val ortSession: OrtSession?,
+    private val result: Result?
+) {
+
+    fun analyze(inputStream: InputStream) {
+        // Step 1: convert image into byte array (raw image bytes)
+        val rawImageBytes = inputStream.readBytes()
+
+        // Step 2: get the shape of the byte array and make ort tensor
+        val shape = longArrayOf(rawImageBytes.size.toLong())
+        val env = OrtEnvironment.getEnvironment()
+        env.use {
+            val inputTensor = OnnxTensor.createTensor(
+                env,
+                ByteBuffer.wrap(rawImageBytes),
+                shape,
+                OnnxJavaType.UINT8
+            )
+            inputTensor.use {
+                // Step 3: call ort inferenceSession run
+                val output = ortSession?.run(Collections.singletonMap("image", inputTensor))
+
+                // Step 4: output analysis
+                output.use {
+                    @Suppress("UNCHECKED_CAST")
```

Comment:
I'm not too familiar with kotlin. why do we need this suppression?

Reply:
this should be auto annotated by kotlin linting. but probably not necessary in this case. removed.